eve/main.yml

```diff
@@ -2671,3 +2686,42 @@ stages:
           name: Trigger multiple-nodes step with snapshot enabled
           stage_names:
             - multiple-nodes
+
+  publish-images:
+    worker: *build_worker
+    steps:
+      - ShellCommand: *wait_for_docker
+      - ShellCommand: *image_registry_login
+      - Git: *git_pull
+      - SetPropertyFromCommand:
+          name: Set shell-ui version as a property
+          property: shell_ui_version
+          command: |
+            . ./VERSION
+            echo "$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH$VERSION_SUFFIX"
+      - ShellCommand:
+          name: Build shell-ui container image
+          command: docker build . --tag shell-ui:v%(prop:shell_ui_version)s
+          workdir: build/shell-ui
+          haltOnFailure: true
```

Comment:
Shouldn't we build (and save) this image during pre-merge instead? So if we want to add tests, we can.